python/tests/test_cb.json

```diff
@@ -157,7 +188,39 @@
                 "#base": [
                     "--cb_explore_adf"
                 ]
-            }
+            },
+            "(",
+            {
+                "--epsilon": [
+                    0.1,
+                    0.2,
+                    0.3
+                ]
+            },
+            "+",
```

Comment:
other exploration algos (--cb_explore_adf_***):
https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/vowpalwabbit/core/src/reductions/cb
Improtant stuff to add:
--squarecb
--synthcover
--regcb
--softmax